lib/portus/json.rb

```diff
@@ -0,0 +1,32 @@
+module Portus
+  # A safe version of the ::JSON class. This class catches exceptions and logs
+  # them properly. This class assumes that the given object is a
+  # request/response.
+  class JSON
+    # Parses the body of the given request/response. Any exception is catched
+    # by this application and it's properly logged.
+    #
+    # Returns the given parsed JSON object on success, nil otherwise.
+    def self.parse(obj)
+      # Depending on whether this is a request or a response, the body will be
+      # a string or an object that responds to :read.
+      body = obj.body.is_a?(String) ? obj.body : obj.body.read
+      ::JSON.parse(body)
+    rescue ::JSON::ParserError => e
+      Rails.logger.warn "JSON: parser error!"
```

Comment:
This piece of code can be executed also by the tasks running inside of Cronos. We don't want to log everything to the rails' log, we should be able to send the messages to the right location.